app/services/creator_service.py

```diff
@@ -188,3 +189,16 @@ async def professional_profile_analytics(request_body: object, params: Optional[
         await executor_event.professional_profile_analytics_event_handler(data=analytics)
 
     return analytics
+
+
+async def get_async_profile_analytics(request_body: object, params: Optional[Dict]) -> Optional[Dict]:
+    async_profile_analytics: Dict = await request_async_profile_analytics(request_body=request_body, params=params)
+
+    if not async_profile_analytics:
+        logging.error(f"Profile Analytics do not exist with requested-filters")
```

Comment:
Let's print the body in the logs, please verify whether it will work or not. If not, try to convert the `request_body` to dict

```suggestion
        logging.error(f"Profile Analytics do not exist with requested-filters: {request_body}")
```